tests/common.py

```diff
@@ -30,6 +30,10 @@
 )
 from homeassistant.auth.permissions import system_policies
 from homeassistant.components import device_automation, persistent_notification as pn
+from homeassistant.components.cloud import (
+    SIGNAL_CLOUD_CONNECTION_STATE,
+    CloudConnectionState,
+)
```

Comment:
I think cloud imports quite a lot of things, importing it here will require every test to import every cloud dependency. Please import these locally in the `async_mock_cloud_connection_status` helper below.

Reply:
dispatcher as well?